dm/templates/dns_managed_zone/dns_managed_zone.py.schema

```diff
@@ -30,9 +30,12 @@ required:
 properties:
   zoneName:
     type: string
+    pattern: ^[a-z]{1}[a-z0-9-]*[a-z0-9]?$
```

Comment:
This expression is incorrect (allows ending dashes). Correct one:
^[a-z]{1}([a-z0-9-]*[a-z0-9])?$

Reply:
Updated example for string limit:
^[a-z]([a-z0-9-]{0,61}[a-z0-9])?$

Reply:
applied